packages/client-features/src/blocks.ts

```diff
@@ -15,7 +14,7 @@ import type { Feature } from '@wp-feature-api/client';
  * Client-side feature to insert a paragraph block.
  */
 export const insertParagraphBlock: Feature = {
-	id: 'core/insert-paragraph-block',
+	id: 'blocks/insert-paragraph-block',
```

Comment:
Not a blocker but it is something that needs to happen as follow-up, we should add isEligible callback that say when an action could be done.
Currently if I'm outside the editor and say "Insert a quote block saying ok" the way try to the inser tool and there is an error "System Error: Failed to execute client feature 'blocks/insert-quote-block'.".